hooks/post/31-mongo_syspath.rb

```diff
@@ -0,0 +1,18 @@
+def install_syspaths
+  `rpm -qa | grep rh-mongodb34-syspaths`
+  if $?.success?
+    logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
+  else
+    logger.info 'rh-mongodb34-syspaths not present, installing.'
+    Kafo::Helpers.execute('yum install -y -q rh-mongodb34-syspaths')
+  end
+end
+
+# Fail if we are not using the Katello/Foreman Proxy Content scenario.
+katello = Kafo::Helpers.module_enabled?(@kafo, 'katello')
+foreman_proxy_content = Kafo::Helpers.module_enabled?(@kafo, 'foreman_proxy_content')
+if katello || foreman_proxy_content
+  install_syspaths
+else
+  logger.warn 'Selected scenario is not correct for rh-mongodb34-syspaths install, skipping'
```

Comment:
Is this really a warning? Remember that `foreman-proxy-certs-generate` also executes this code. That means you will fill logs with warnings that say nothing. IMHO this is at most a debug.